test/index.spec.js

```diff
@@ -63,20 +63,6 @@ describe('multihashing', () => {
   })
 
   describe('invalid arguments', () => {
-    it('returns an error on non implemented func', (done) => {
-      multihashing(new Buffer('beep boop'), 'blake2b', (err) => {
```

Comment:
I'm not going to do that for now, I would have to use some requiere mocking tool to change the content of `multihashes`, to add another function